content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -246,7 +245,7 @@ Windows has strict compatibility rules, where the host OS version must match the
 
 ##### Feature Restrictions
 
-* TerminationGracePeriod: not implemented
+* TerminationGracePeriod: requires CRI-containerD
 * Single file mapping: to be implemented with CRI-ContainerD
```

Comment:
I think the Windows OS fixes for correct single file mount on Windows are widely available. 
Let me confirm and i'll update the docs here.

Reply:
The Windows OS fixes are in place but we need to make an update in K8s which we'll do in v1.21